packages/rich-text/src/RichTextEditor.mdx

```diff
@@ -61,7 +61,16 @@ import { RichTextEditor } from '@contentful/field-editor-rich-text';
     };
 
     const initialValue = JSON.parse(window.localStorage.getItem('initialValue')) || undefined
-    const [field, mitt] = useMemo(() => createFakeFieldAPI(mock => mock, initialValue), []);
+    const fieldValidations = JSON.parse(window.localStorage.getItem('fieldValidations')) || undefined
```

Comment:
Should this default to undefined and not []?

Reply:
I'm falling back to undefined as I think local storage uses null by default. Then later on it will fallback to whatever value the mock gives us if it has one. I don't want to hard code this to []